array/DNAm/preprocessing/QC.rmd

```diff
@@ -108,11 +112,11 @@ remove_unwanted_columns <- function(columns, QCmetrics) {
   return(columns[columns_to_keep])
 }
 
-define_plot_parameters <- function(QCmetrics, techVar, bioVar) {
-  potential_plot_columns <- c(techVar, bioVar)
-  plot_columns <- remove_unwanted_columns(potential_plot_columns, QCmetrics)
+define_plot_parameters <- function(QCmetrics, projVar) {
+  plot_columns <- c(remove_unwanted_columns(projVar, QCmetrics), "Sentrix_ID", "Sentrix_Position")

```

Comment:
Is there any chance that `Sentrix_ID` or `Sentrix_Position` could be columns that would be removed by `remove_unwanted_columns`? I'm pretty sure the only problem could come from having the same value in every row for these columns (which I doubt will ever happen). 
Should be fine.

Reply:
They could probably be all different values (all entries differ) in some datasets (which would mean they would have previously been removed). This check in `remove_unwanted_columns` was put in place to avoid having variables plotted that don't give the user any extra information.

But I think you said you always want these two on the plots anyways, so this shouldn't matter.

Reply:
Agreed